config/webpack.config.js

```diff
@@ -149,12 +149,12 @@ async function getScaffoldAssets() {
  * Generates HTML files for each app and widget.
  *
  * @param {String} buildPath - Path to the overall build destination.
+ * @param {Object} scaffoldAssets - Map of scaffold asset filenames to file contents.
  *
  * @return {HtmlWebpackPlugin[]} - Array of HtmlWebpackPlugin instances,
  *   representing the HTML files to generate for each app and widget.
  */
-async function generateHtmlFiles(buildPath) {
-  const scaffoldAssets = await getScaffoldAssets();
+function generateHtmlFiles(buildPath, scaffoldAssets) {
```

Comment:
I refactored this function so that we won't need to fetch the scaffold assets twice, since we'll be doing that later in the config.